lib/salus/repo.rb

```diff
@@ -95,5 +95,18 @@ class Repo
     def initialize(path_to_repo = nil)
       @path_to_repo = path_to_repo
     end
+
+    ##
+    # @param [Array] args CLI rg command and options to run
+    # @return [Array<String>] Relative path of files returned
+    # from running the command.
+    def run_rg(*args)
+      data = IO.popen(args, chdir: @path_to_repo).read
```

Comment:
I'm not seeing `chdir` as a valid option for `IO.popen`.  It is most likely has no effect in the call.
https://docs.ruby-lang.org/en/master/IO.html#method-c-popen

Please see the links under `Optional keyword arguments opts specify:`

Reply:
If. you want to run this from the right directory, you can do
```ruby
      data = ''
      Dir.chdir(path_to_repo) do
        data = IO.popen(args).read	
      end
```